docs/docs/legal/rate-limit-policy.md

```diff
@@ -23,15 +23,9 @@ You should consider to implement [exponential backoff](https://en.wikipedia.org/
 
 ## What rate limits do apply
 
-### Login, Register, Reset Limits
-
-For the rate limits of the Login, Register and Reset features please visit [Login Rate Limits](/docs/apis/ratelimits/login)
-
-### API Rate Limits
-
-For our API rate limits please check the [API Endpoint Rate Limits](/docs/apis/ratelimits/api)
+For ZITADEL Cloud, we have a rate limiting rule for login paths (login, register and reset features) and for API paths each. Learn more about [the exact rules](/docs/apis/ratelimits).
 
 ## Load Testing
 
-If you would like to conduct load testing of ZITADEL Cloud or a managed instance, you MUST request to do so with a minimum of 2 weeks notice before the test by contacting us at support@zitadel.com.  
+If you would like to conduct load testing of ZITADEL Cloud or a managed instance, you MUST request to do so with a minimum of 2 weeks notice before the test by contacting us at support@zitadel.com.
```

Comment:
@mffap can we change the legal section when it is just language/style like this?

Reply:
Thanks for being mindful about this :)
Cosmetic changes are generally no issue as long as they won't change the meaning.